### ISSUE TITLE
[CY-3362] Is there a config option to limit the number of "New" issues to 0? Rather than failing if the total issues are >0?

### ISSUE THREAD

**Issue by chrisblakley** [codacy/codacy-analysis-cli-action#16]
I got my first action to run and it resulted it thousands of warnings, errors, and info. I am aware that there are some ongoing issues on this repo, but I don't want a new PR submitted to my repo to fail because there are lots of existing issues already...

Here is what Codacy itself looks like– everything is great– no new issues:
![Screen Shot 2020-12-15 at 4 45 20 PM](https://user-images.githubusercontent.com/6349678/102276640-14b13700-3ef5-11eb-96bb-ce1da6961ce7.png)

But then the Github Action is saying the commit fails because it found tens of thousands of warnings/errors:
![Screen Shot 2020-12-15 at 4 45 50 PM](https://user-images.githubusercontent.com/6349678/102276688-28f53400-3ef5-11eb-83d4-7c6e92dcccca.png)

I see in [this documentation](https://github.com/codacy/codacy-analysis-cli) that there is a setting for `--max-allowed-issues` which fails at 0 by default, but I don't see a configuration option to set a maximum number of "New" issues...

Basically, I want to scan pull requests to make sure they aren't adding any **new** issues to the codebase. Is that possible?

Maybe I'm misunderstanding something, but it seems like every PR that is submitted to my repo is going to fail because of code that was written years ago and has not been addressed yet, or for Codacy rules that I have not yet configured on the existing code.

Could I get some clarification on this and also I'll need some guidance as to how to implement configurations into my yaml workflow (if possible). Thank you.

**Comment by franciscodua** [comment]
Hi @chrisblakley 
The action has an option for that called `max-allowed-issues`.

Your action can be something like this:
```yaml
name: Codacy

on:
  push:
    branches: [ main ]

jobs:
  Scan:
    runs-on: ubuntu-latest
    steps:
    - uses: actions/checkout@v2
    - name: Codacy Analysis
      uses: codacy/codacy-analysis-cli-action@master
      with:
        project-token: ${{ secrets.CODACY_PROJECT_TOKEN }}
        max-allowed-issues: 500
```

Does this work for you?

**Comment by chrisblakley** [comment]
Thanks for your response. Does the `max-allowed-issues` parameter refer to the total number of issues in the repo before it fails, or the number of **new** issues specifically? The repo has 11k+ issues during a Codacy scan so I want to avoid every new PR from failing because of unrelated issues already in the codebase.

**Comment by franciscodua** [comment]
Hi @chrisblakley 

The CLI always "finds" every issue.
But if you want to know what issues were added in the last commit you can use Codacy. You can still run the CLI and upload the results to Codacy.

Check this [page](https://docs.codacy.com/repositories/quality-settings/) for more information on quality settings.

Let us know if this answered your question

**Comment by chrisblakley** [comment]
Thanks, I am using Codacy, but I was hoping this action would be useful for contributors to my repo to automatically check their PRs without having access to view my Codacy account.

I understand if that is not a feature of this service. Thanks for your time.
